.github/workflows/precompiled.yaml

```diff
@@ -319,9 +319,12 @@ jobs:
           driver_branch_json="${{ needs.set-driver-version-matrix.outputs.driver_branch }}"
           DRIVER_BRANCHES=($(echo "$driver_branch_json" | jq -r '.[]'))
           echo "DRIVER_BRANCHES=${DRIVER_BRANCHES[*]}" >> $GITHUB_ENV
+      - name: Set kernel version in holodeck_${{ env.DIST }}.yaml
+        run: |
+          yq e '.spec.kernel.version = "${{ env.KERNEL_VERSION }}"' -i tests/holodeck_${{ env.DIST }}.yaml
```

Comment:
Instead of doing a variable substitution? Can we append the kernelVersion field to the yaml? I think `yq` would support this